### PR TITLE
Simplify doc for release

### PR DESCRIPTION
**Pull request by lbartoletti**
Remove wget and QWAT_VERSION and add a link to the release page.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -122,6 +122,7 @@ Optionally, you can restore a sample dataset. For that you need to download the
 
 Second way. Create a database from the sample dump using ``pg_restore``
 ```````````````````````````````````````````````````````````````````````
+QWAT can be tested by downloading the QGS project repository `here <https://github.com/qwat/QWAT>`_ and restoring the latest `data_and_structure_sample.backup <https://github.com/qwat/qwat-data-model/releases/latest>`_ in PostgreSQL (+ extension PostGIS and hstore).
```
Comment (elemoine) [inline README.rst]: I don't find this very clear. Does that mean the full QWAT Git repo should be cloned? Or does it mean that only the QGIS project (the `.qgs` file) should be downloaded?

Also, I'd remove `(+ extension PostGIS and hstore)`, which is a bit confusing in my opinion.

File: doc/demo-guide/index.rst [hunk 2]
```diff
@@ -1,7 +1,18 @@
 Demo data and project
 =====================
 
-QWAT can be tested by downloading the QGS project repository `here <https://github.com/qwat/QWAT>`_ and restoring the latest `full data dump <https://github.com/qwat/qwat-data-model/releases>`_ in PostgreSQL (+ extension PostGIS and hstore).
+QWAT can be tested by downloading the QGS project repository `here <https://github.com/qwat/QWAT>`_ and restoring the latest `data_and_structure_sample.backup <https://github.com/qwat/qwat-data-model/releases/latest>`_ in PostgreSQL (+ extension PostGIS and hstore).
+
+On the download page you can find theses backups:
+
+- `qwat_vX.X.X_data_and_structure_sample.backup`: Restore a full demo with dataset sample
```
Comment (elemoine) [inline doc/demo-guide/index.rst]: Suggestion : "Restore the QWAT schema plus the sample dataset"

File: doc/demo-guide/index.rst [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
+
+- `qwat_vX.X.X_structure_only.backup`:  Restore only the structure of QWAT (you need at least demo or custom value_list)
```
Comment (elemoine) [inline doc/demo-guide/index.rst]: Suggestion: "Restore the QWAT schema only"

I find the sentence "you need at least demo or custom value_list" a bit confusing. What is "demo" for example? Instead I think I'd say: "in addition you'll also need to restore the value_list_data_only dump".

File: doc/demo-guide/index.rst [hunk 4]
```diff
@@ -1,7 +1,18 @@
...
+
+- `qwat_vX.X.X_value_list_data_only.backup`: Restore the value list of QWAT demo project
+
+- `qwat_vX.X.X_data_only_sample.backup`: Restore the dataset sample (must have run init_qwat or restore the structure of QWAT)
```
Comment (elemoine) [inline doc/demo-guide/index.rst]: Suggestion: "Restore the sample dataset (the `init_qwat.sh` script should be run prior to restoring the sample dataset)"

File: README.rst (hunk 1 above)
Comment (lbartoletti) [reply README.rst]: For now, you have to download the repo (git or zip) to open the QGS project.
Maybe we should create a dedicated repository for that?

File: doc/demo-guide/index.rst (hunk 2 above)
Comment (lbartoletti) [reply doc/demo-guide/index.rst]: Done.

File: doc/demo-guide/index.rst (hunk 3 above)
Comment (lbartoletti) [reply doc/demo-guide/index.rst]: Right. Done

File: doc/demo-guide/index.rst (hunk 4 above)
Comment (lbartoletti) [reply doc/demo-guide/index.rst]: Right. Done.

**Pull request merged**